ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -100,6 +100,28 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# Link Tracking Groups #}
+{%     set trk_grps = namespace() %}
+{%     set trk_grps.eth_intfs = [] %}
```

Comment:
We don't need namespace here because we are appending the list.
so, set trk_grps_eth_ints = [] and then .append to this list should work.

Reply:
I will refactor this a bit to make the code more efficient.

Reply:
It's funny that I am reviewing my own code.

Reply:
Not much I could change. I removed the namespace and make the variable names more clear.